pyccel/parser/semantic.py

```diff
@@ -1139,6 +1143,67 @@ def _handle_function(self, expr, func, args, *, is_method = False, use_build_fun
 
             return new_expr
 
+    def _handle_kernel(self, expr, func, args):
+        """
+        Create the node representing the kernel function call.
+
+        Create a FunctionCall or an instance of a PyccelInternalFunction
+        from the function information and arguments.
+
+        Parameters
+        ----------
+        expr : IndexedFunctionCall
+               Node has all the information about the function call.
+
+        func : FunctionDef |Interface | PyccelInternalFunction type
+               The function being called.
+
+        args : tuple
+               The arguments being passed to the function.
+
+        Returns
+        -------
+        Pyccel.ast.cuda.KernelCall
+            The semantic representation of the kernel call.
+        """
+        if(len(expr.indexes) != 2):
+            errors.report(INVALID_KERNEL_LAUNCH_CONFIG,
+                    symbol=expr,
+                    severity='fatal')
+        if(len(func.results)):
```

Comment:
In Python we usually don't use parentheses unless they help you understand how conditions are grouped. Specifically we almost never use then around conditions in an if statement. This is not C.

```suggestion
        if len(expr.indexes) != 2:
            errors.report(INVALID_KERNEL_LAUNCH_CONFIG,
                    symbol=expr,
                    severity='fatal')
        if len(func.results):
```

Reply:
[1f19376](https://github.com/pyccel/pyccel-cuda/pull/42/commits/1f19376d9e255f35d728a9b5d34e687b4b37f3c2)